docs/src/tools.md

```diff
@@ -0,0 +1,41 @@
+# Tools
+
+This section will cover setup of various tools that can help you with development of tt-forge-fe.
+
+## Pre-commit
```

Comment:
Can we include formatters as pre-commit checks as well (if not already there :))? By [docs](https://pre-commit.com/hooks.html), seems like it's possible to hook them, but didn't try it out. 

@vmilosevic it'll be good to add formaters and pre-commit as part of docker :)) What do you think?

Reply:
Not sure I follow? The formatters are included as pre-commit checks, see `.pre-commit-config.yaml`...

Reply:
The `pre-commit` tool takes the config file (`.pre-commit-config.yaml`) and installs appropriate git hooks. End result is that those hooks will be executed whenever you run `git commit`.

Reply:
Thanks, that answers my question! 

In sum, I didn't saw that it was a part of the pre-commit config file.. All good! Thanks for the explanation.